### ISSUE TITLE
questions on differences between code and paper

### ISSUE THREAD

**Issue by turmeric-blend** [chaoshangcs/GTS#4]
Hi, I noticed in your paper you mentioned:

>  In practice, we anneal s (temperature) progressively in training such that it tends to zero.

> vary the dropout rate {0.1, 0.2, 0.3} in hyperparameter search.

Hwoever, in the code, [temperature](https://github.com/chaoshangcs/GTS/blob/208bc8bb5afbbdd1ea6cddbecbb9ccd0e9effdc7/model/pytorch/supervisor.py#L258) was not decayed and  [dropout](https://github.com/chaoshangcs/GTS/blob/208bc8bb5afbbdd1ea6cddbecbb9ccd0e9effdc7/model/pytorch/model.py#L142) was not used.

Is it during experimentation, you found out it was not necessary to decay temperature and dropout not needed, to achieve the best results?

**Comment by chaoshangcs** [comment]
I removed some optional rows when I reorganize the code. Now I have added temperature and dropout to the code again as options.

**Comment by turmeric-blend** [comment]
thanks.
